spec/lib/os_places_api/client_spec.rb

```diff
@@ -177,7 +189,13 @@
           .to_return(status: 200, body: successful_response.to_json)
 
         expect(Postcode).not_to receive(:create!)
-        expect(client.locations_for_postcode(postcode)).to eq([location])
+        expect(client.locations_for_postcode(postcode)).to eq(
+          {
+            "average_latitude" => 51.5144547,
+            "average_longitude" => -0.0729933,
```

Comment:
I would have a `let(:average_latitude) { 51.5144547 }` further up in the file, and remove the duplication in this file.

Reply:
👍 I like this. Thanks for recommendation.